Source/TailwindTraders.Mobile/TailwindTraders.UITests/AppManager.cs

```diff
@@ -5,9 +5,12 @@ namespace TailwindTraders.UITests
 {
     internal static class AppManager
     {
-        private const string ApkPath = "../../../Binaries/com.microsoft.tailwindtraders-Signed.apk";
-        private const string AppPath = "../../../Binaries/TailwindTraders.Mobile.iOS.app";
-        private const string IpaBundleId = "com.microsoft.TailwindTraders-df";
+        private const string UITestAndroidPath = "../../../TailwindTraders.Mobile.Android/bin/UITEST/";
+        private const string ApkPath = UITestAndroidPath + "com.microsoft.tailwindtraders-Signed.apk";
+
+        private const string UITestiOSPath = "../../../TailwindTraders.Mobile.iOS/bin/iPhone/UITEST/";
+        private const string AppPath = UITestiOSPath + "TailwindTraders.Mobile.iOS.app";
+        private const string IpaBundleId = "TailwindTraders.Mobile.iOS.ipa";
```

Comment:
I think there's something wrong here: IpaBundleId refers to the actual file name instead of the bundle id, which I'd expect from the var name